python-qa-tools/load-test/multi_msg_load.py

```diff
@@ -0,0 +1,103 @@
+import argparse, os, sys, time
+from modules.bank.query import query_balances
+from utils.bank import print_balance_deductions
+from utils.commands import exec_command
+from utils.keys import fetch_account_address
+from utils.txs import fetch_seq_no, signed_tx, unsigned_tx, write_json
+from utils.types import account_type, num_txs_type
+
+DAEMON = os.getenv('DAEMON')
+DENOM = os.getenv('DENOM')
+CHAINID = os.getenv('CHAINID')
+DAEMON_HOME = os.getenv('DAEMON_HOME')
+HOME = os.getenv('HOME')
+
+parser = argparse.ArgumentParser(description='This program takes inputs for intializing multi message load test.')
+parser.add_argument('FROM', type= account_type, help= 'From which account the transaction should be intialized')
+parser.add_argument('TO', type= account_type, help= 'Reciever account number.')
+parser.add_argument('NUM_TXS', type = num_txs_type, help= 'Number of transactions to be made, Min. should be 1000')
+
+args = parser.parse_args()
+FROM, TO, NUM_TXS = int(args.FROM), int(args.TO), int(args.NUM_TXS)
+
+if FROM == TO:
+    sys.exit('Error: The values of arguments "TO" and "FROM" are equal make sure to set different values')
+
+RPC, num_msgs = "http://127.0.0.1:16657", 30
+
+#### Fetching Bech32 addresses ######
+
+acc1, acc1err = fetch_account_address(f"account{FROM}")
```

Comment:
```suggestion
acc1, acc1err = fetch_account_address_from_keyname(f"account{FROM}")
```
But we don't need this anyways